lib/features/school_event/screens/school_event_info_screen.dart

```diff
@@ -72,7 +72,7 @@ class SchoolEventInfoScreen extends StatelessWidget {
                                 ),
                                 const SizedBox(height: 5),
                                 Text(
-                                  'Thanks to the generous support of the Bank of Oklahoma, we are delighted to give you:',
+                                  'Thanks to the generous support of the Bank of Oklahoma to Mark & Debbie Plaster & Dr. Presley and Associates, we are delighted to give you:',
```

Comment:
I think Andy's idea was to replace Bank of Oklahoma by Mark ......... Associates


Andy's message was:
"So **Bank of Oklahoma** needs to be changed to **Mark & Debbie Plaster & Dr. Presley and Associates**"